index.html

```diff
@@ -207,17 +207,21 @@ <h4>Skills</h4>
                         </ul>
                     </div>
                 </li>
-            </ul>
-            
+            </ul>        
+        </section>
+
+        <section id="contact-form">
```

Comment:
- [ ] [OPTIONAL] Since your major work is done, how about sparing some time to add the last part of the figma design to yours so that it can match it even well, thus improving the look of your beautiful page? 👇 
![image](https://user-images.githubusercontent.com/99715640/213055126-9ae84dd2-802e-449e-aa71-1f56aeea500d.png)